static/js/components/AutoComplete.js

```diff
@@ -393,7 +405,8 @@ class AutoComplete extends Component {
       triggerUpdateOnFocus, // eslint-disable-line no-unused-vars
       openOnFocus, // eslint-disable-line no-unused-vars
       maxSearchResults,
-      dataSource,
+      dataSource, // eslint-disable-line no-unused-vars
+      menuHeight,
```

Comment:
that's a concern of the child component. i still think height should be part of menuStyle (as 'maxHeight'). Menu.js can worry about passing the correct value for 'menuHeight'